packages/whenable/src/message-breakpoints.js

```diff
@@ -0,0 +1,179 @@
+import { getEnvironmentOption } from '@endo/env-options';
```

Comment:
I diffed, and as I hoped, this is the same as message-breakpoints.js in @endo/eventual-send. Could you just import from that one instead, rather than duplicating it?

Reply:
Understood. We can address this well after this PR is merged. Possibly after moving whenable (or vow) to endo. Resolving.